.github/workflows/opencensus.yaml

```diff
@@ -0,0 +1,66 @@
+name: opencensus
+on:
+  push:
+    branches:
+    - main
+    paths:
+    - 'opencensus/**'
+  pull_request:
+    paths:
+    - 'opencensus/**'
+  pull_request_target:
+    types: [labeled]
+  schedule:
+  - cron:  '0 2 * * *'
+jobs:
+  test:
```

Comment:
maybe use `opencensus` instead of `test` as the job name. Then make `opencensus` a mandatory test.